### PR TITLE
(DPS-3203) Customer SSO versioning

### PR DESCRIPTION
**Pull request by jakubsikora**
# ❗️ Feature branch / Deploy Preview

We suggest to name your branch with `feature/`, for example, `feature/DPS-2947`. See below notifications from **Netlify** about the url.

# 🚀 Links

Replace the placeholder data in links 👇

- [Jira](https://livechatinc.atlassian.net/browse/DPS-3203)

# 📓 Description
Add support for versions inside redoc docs.

### CODE REVIEW

File: src/pages/authorization/customer-accounts-api/index.mdx [hunk 1]
```diff
@@ -5,6 +5,7 @@ subcategory: "customer-accounts-api"
 title: Customer Accounts API
 tagline: "Find out how you to configure chat webhooks."
 desc: "Use chat webhooks to get notified about the events happening during chats, for example, when there is an incoming chat, when someone sends a message, or when an agent tags a chat."
+apiVersion: "3.3"
```
Comment (ale-kacperczyk) [inline src/pages/authorization/customer-accounts-api/index.mdx:8]: This is supposed to be 1.0

File: src/pages/authorization/customer-accounts-api/v2.0/index.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+weight: 30
+category: "authorization"
+subcategory: "customer-accounts-api"
+title: Customer Accounts API
+tagline: "Find out how you to configure chat webhooks."
+desc: "Use chat webhooks to get notified about the events happening during chats, for example, when there is an incoming chat, when someone sends a message, or when an agent tags a chat."
+apiVersion: "2.0"
```
Comment (ale-kacperczyk) [inline src/pages/authorization/customer-accounts-api/v2.0/index.mdx:8]: 2.0 is stable

File: src/constant/index.js [hunk 3]
```diff
@@ -20,6 +20,12 @@ const VERSIONS_GROUPS = {
     DEPRECATED_VERSIONS: ["3.1"],
     ALL_VERSIONS: ["3.4", "3.3", "3.2", "2.0"],
   },
+  "customer-accounts-api": {
+    STABLE_VERSION: "3.3",
+    LEGACY_VERSIONS: ["2.0"],
+    DEV_PREVIEW_VERSION: "",
+    ALL_VERSIONS: ["3.3", "2.0"],
```
Comment (ale-kacperczyk) [inline src/constant/index.js:27]:
```suggestion
    ALL_VERSIONS: ["1.0", "2.0"],
```

File: src/constant/index.js [hunk 4]
```diff
@@ -20,6 +20,12 @@ const VERSIONS_GROUPS = {
     DEPRECATED_VERSIONS: ["3.1"],
     ALL_VERSIONS: ["3.4", "3.3", "3.2", "2.0"],
   },
+  "customer-accounts-api": {
+    STABLE_VERSION: "3.3",
+    LEGACY_VERSIONS: ["2.0"],
```
Comment (ale-kacperczyk) [inline src/constant/index.js:25]:
```suggestion
    LEGACY_VERSIONS: ["1.0"],
```

File: src/constant/index.js [hunk 5]
```diff
@@ -20,6 +20,12 @@ const VERSIONS_GROUPS = {
     DEPRECATED_VERSIONS: ["3.1"],
     ALL_VERSIONS: ["3.4", "3.3", "3.2", "2.0"],
   },
+  "customer-accounts-api": {
+    STABLE_VERSION: "3.3",
```
Comment (ale-kacperczyk) [inline src/constant/index.js:24]:
```suggestion
    STABLE_VERSION: "2.0",
```

File: src/components/Page/index.js [hunk 6]
```diff
@@ -180,32 +182,38 @@ const Page = ({ frontMatter, children }) => {
...
+                    <LeftColumnRedoc>
+                      <NavHeader>
+                        <Link href={"/"} style={{ color: "inherit" }}>
+                          <span>
+                            <HomeIcon width={18} style={{ display: "block" }} />
+                          </span>
+                        </Link>
+                        <ChevronRight width={14} />
+                        <span
+                          style={{
```
Comment (arturfracala) [inline src/components/Page/index.js:204]: I believe we don't use inline styles anymore 🙈

File: src/utils/index.js [hunk 7]
```diff
@@ -116,11 +116,17 @@ export const setupDocsearch = () => {
 };
 
 export const getStableVersion = (category) => {
+  console.log(category);
```
Comment (arturfracala) [inline src/utils/index.js:119]: Please remove this

**Review by arturfracala — CHANGES_REQUESTED** [review]
Nice work 👊 Needs some polishing

**Review by ale-kacperczyk — APPROVED** [review]

**Review by arturfracala — APPROVED** [review]

**Pull request merged**